lib/rb_shift/client.rb

```diff
@@ -62,11 +64,16 @@ def projects(update = false)
     end
 
     def execute(command, **opts)
-      `oc --server="#{@url}" --token="#{@token}" #{command}  #{unfold_opts opts}`
+      command = "oc --server=\"#{@url}\" --token=\"#{@token}\" #{command}  #{unfold_opts opts}"
+      Open3.popen3 command do |_, _, stderr, wait_thrd|
+        err_code = wait_thrd.value
+        raise InvalidCommandError unless stderr.gets.nil? && err_code.success?
```

Comment:
Should non empty stderr lead to ICE even though the status code indicates success?